ts/RootContainer.tsx

```diff
@@ -137,7 +151,8 @@ class RootContainer extends React.PureComponent<Props> {
 
 const mapStateToProps = (state: GlobalState) => ({
   deepLinkState: state.deepLink,
-  isDebugModeEnabled: state.debug.isDebugModeEnabled
+  isDebugModeEnabled: state.debug.isDebugModeEnabled,
+  backendInfo: state.backendInfo.serverInfo
```

Comment:
could you create a selector instead of direct accessing the state? (**_serverInfoData_**)